packages/jest-matchers/src/matchers.js

```diff
@@ -312,7 +314,11 @@ const matchers: MatchersObject = {
     return {message, pass};
   },
 
-  toBeCloseTo(actual: number, expected: number, precision?: number = 2) {
+  toBeCloseTo(
+    actual: number,
+    expected: number,
+    precision?: number = 2,
```

Comment:
Why did you make this change?

It made me realize that this won't work in node 4. Since you already touched it, can you change it to not use default values and instead inline this like

```
if (precision == null) {
  precision = 2;
}
```